num2/num2/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+class Program
+{
+    static void Main(string[] args)
+    {
+        Console.WriteLine("Введите координату x координата центра квадрата");
+        Console.WriteLine("-------------------------");
+        double x1 = Convert.ToDouble(Console.ReadLine());
+        Console.WriteLine("-------------------------");
+        Console.WriteLine("Введите координату y координата центра квадрата");
+        Console.WriteLine("----------------------------");
+        double y1 = Convert.ToDouble(Console.ReadLine());
+        Console.WriteLine("----------------------------");
+        Console.WriteLine("Введите координату x координата вершины");
+        Console.WriteLine("----------------------------");
+        double x2 = Convert.ToDouble(Console.ReadLine());
```

Comment:
См. комментарий выше
```suggestion
        double x2 = double.Parse(Console.ReadLine());
```